### PR TITLE
Add OnlineFriendsViewController

### PR DESCRIPTION
**Pull request by Chuongv**
User can pick which friend to call
https://github.com/Antidote-for-Tox/objcToxBot/issues/2

### CODE REVIEW

File: objcToxBot/CallAudioTask.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//
+//  Created by Chuong Vu on 9/4/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "TaskProtocol.h"
+
+@interface CallAudioTask : NSObject <TaskProtocol>
```
Comment (dvor) [inline objcToxBot/CallAudioTask.h:17]: Can you please add comment about what this task will do (you can check other tasks as example)?

File: objcToxBot/OnlineFriendsViewController.m [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    [self setupConstraints];
+}
+
+- (void)createBlurrView
+{
+    UIBlurEffect *effect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleDark];
+
+    self.visualView = [[UIVisualEffectView alloc] initWithEffect:effect];
+    self.visualView.frame = self.view.frame;
+    [self.view addSubview:self.visualView];
```
Comment (dvor) [inline objcToxBot/OnlineFriendsViewController.m:42]: Now you like to add blur effect everywhere :wink:

Comment (Chuongv) [inline objcToxBot/OnlineFriendsViewController.m:42]: Once you use it once, you have to use it all the time! :laughing:

**Comment by dvor** [comment]
Cool, thank you!

**Pull request merged**